libs/utils/utils.js

```diff
@@ -420,6 +420,27 @@ export function decorateSVG(a) {
   }
 }
 
+function hasUrlWithPipe(str) {
+  const regex = /^https?:\/\/.*\|.*/;
+  return regex.test(str);
+}
+
+export function decorateImageLinks(a) {
+  const images = a.querySelectorAll('img');
+  if (!images.length) return a;
+  [...images].forEach((img) => {
+    if (!img.alt || !(hasUrlWithPipe(img.alt))) return;
+    const [source, alt] = img.alt.split('|');
+    const textUrl = new URL(source.trim());
+    if (alt && alt.trim().length > 0) img.alt = alt.trim();
```

Comment:
Nit: you could remove the `> 0` part, if the length is 0, it will be falsy anyway.

Reply:
Good call ✔️